pkg/server/sandbox_run.go

```diff
@@ -78,9 +78,47 @@ func (c *criContainerdService) RunPodSandbox(ctx context.Context, r *runtime.Run
 	if err != nil {
 		return nil, fmt.Errorf("failed to get sandbox image %q: %v", c.sandboxImage, err)
 	}
+	//Create Network Namespace if it is not in host network
+	hostNet := config.GetLinux().GetSecurityContext().GetNamespaceOptions().GetHostNetwork()
```

Comment:
Please add a comment to explain that if not using host network, sandbox.NetNSPath and sandbox.NetNS will be left empty, we don't and shouldn't use it.